docs/storage.md

```diff
@@ -179,6 +189,38 @@ Now you can create the SR where `server` is your mon ip.
 * You may specify `serverport` option if you are using any other port than 6789.
 :::
 
+### Ceph iSCSI gateway
+
+This is at this moment the only way to connect to Ceph with no modifications of dom0, it's possible to create multiple Ceph iSCSI gateways following this: https://docs.ceph.com/docs/master/rbd/iscsi-target-cli/
```

Comment:
User says "no modifications of dom0" but I see various packages that need to be installed in the linked docs.

Reply:
Ceph iSCSI gateway sits outside dom0, probably another Virtual or Physical machine. The packages referred are to be installed on iSCSI gateway node. For XCP-ng dom0, no modifications are needed as it would use LVMoISCSISR driver to access the iSCSI LUN presented by these gateways.

Reply:
Also if it requires external packages, it should link to https://xcp-ng.org/docs/additionalpackages.html with a word of warning

Reply:
Understood. Then please add a sentence that explains that :)

Reply:
Added that sentence to better explain it.